lib/sparql-queries.ts

```diff
@@ -122,85 +118,10 @@ export async function executeDeleteQuery (quads: RDF.Quad[]) {
   }
 }
 
-export async function getLatestTimestamp (baseResource: RDF.NamedNode, treeProperties: TreeProperties) {
```

Comment:
Versioning (and only emitting newer versions) now happens by the underlying library, so we don't need to fetch a member's latest timestamp anymore.

Reply:
I think this still allows some performance gains (fewer inserts) in case you don't have a state file but do already have data (for example when upgrading the consumer), but am fine with keeping things simple for now.